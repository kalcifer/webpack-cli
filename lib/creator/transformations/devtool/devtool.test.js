const defineTest = require('../../../transformations/defineTest');

defineTest(__dirname, 'devtool', 'devtools-0', {devtool : 'source-map'});
defineTest(__dirname, 'devtool', 'devtools-1', {devtool : 'source-map'});
