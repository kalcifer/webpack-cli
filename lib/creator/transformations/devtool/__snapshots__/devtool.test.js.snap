// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`devtool transforms correctly using "devtools-0" data 1`] = `
"module.exports = {
    entry: 'index.js',
    output: {
		filename: 'bundle.js'
	},

    devtools: 'source-map'
}
"
`;

exports[`devtool transforms correctly using "devtools-1" data 1`] = `
"module.exports = {
	entry: 'index.js',
	output: {
		filename: 'bundle.js'
	},
	devtools: 'source-map'
}
"
`;
